kotlinpoet/src/main/java/com/squareup/kotlinpoet/CodeBlock.kt

```diff
@@ -58,7 +58,8 @@ import kotlin.reflect.KClass
  */
 public class CodeBlock private constructor(
   internal val formatParts: List<String>,
-  internal val args: List<Any?>
+  internal val args: List<Any?>,
+  internal val emittedFunNames: List<String> = emptyList()
```

Comment:
`CodeBlock` is at a lower level of abstraction than `FunSpec`, so I think it's incorrect to have `CodeBlock` know about function names.